### PR TITLE
Adding questions file.  Updating Account to be proper class.

### PR DESCRIPTION
**Pull request by adam-in-ict**
Adding questions file that was causing issues with creating deployment project for Zendesk (INT-2151. 

Updating Account entity based on feedback from a previous similar ingestion (Zoom).

### CODE REVIEW

File: jupiterone/questions/questions.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+sourceId: managed:zendesk
+integrationDefinitionId: '${integration_definition_id}'
+questions:
+  - id: managed-question-zendesk-access-list
+    title: Who has access to Zendesk in my organization?
+    description: Returns a list of users with Zendesk access.
+    queries:
+      - name: good
+        query: FIND zendesk_user AS zu RETURN zu.displayName AS "Name", zu.username AS "Username", zu.id as "ID"
```
Comment (austinkelleher) [inline jupiterone/questions/questions.yaml:10]: Let's remove the quotes around the aliases. I didn't even know that was supported!

File: src/steps/account/__recordings__/fetchAccount_3692983622/recording.har [hunk 2]
```diff
@@ -45,19 +45,19 @@
...
+            "size": 362,
+            "text": "[\"1f8b080000000000000355cdbd0e82401004e077d91a811090789dad85d656e4b85be209774bee470c847777136d2c67f2656603a9142517416c90fc04021e31ce4114c533cd26a22787078daf7c45a7318cb9220b19386991ede56b6e0eb90bd2e99ede20063905e49c7a4d561ac7f07f8c713416bb81bc957c5dd5bf62e537d6678bde28595c71e9eee447f6b438f49dd120eaaa6c9be6d8b6e5a9cac0a6299ad9934e8a87a24fb8ef1f4a49d726d5000000\"]"
```
Comment (austinkelleher) [inline src/steps/account/__recordings__/fetchAccount_3692983622/recording.har:60]: Looks like these recording response text values are gzip'd. We should leverage the `unzipGzippedRecordingEntry` recording mutation in the SDK. Example: https://github.com/JupiterOne/graph-datadog/blob/main/test/recording.ts#L14

**Review by austinkelleher — CHANGES_REQUESTED** [review]
Thanks for the changes! Made a few comments.

File: jupiterone/questions/questions.yaml (hunk 1 above)
Comment (adam-in-ict) [reply jupiterone/questions/questions.yaml:10]: Yep!  I believe you can use quotes to include spaces in your aliases.  But I agree that it's not needed in this instance.

File: src/steps/account/__recordings__/fetchAccount_3692983622/recording.har (hunk 2 above)
Comment (adam-in-ict) [reply src/steps/account/__recordings__/fetchAccount_3692983622/recording.har]: Updated in latest commit.

File: test/recording.ts [hunk 3]
```diff
@@ -11,5 +12,8 @@ export function setupZendeskRecording(
 ): Recording {
   return setupRecording({
     ...input,
+    redactedRequestHeaders: ['Authorization'],
+    redactedResponseHeaders: ['set-cookie'],
```
Comment (austinkelleher) [inline test/recording.ts:16]: Interesting. What was the motivations of adding `redactedResponseHeaders` and `redactedRequestHeaders`?

**Review by austinkelleher — APPROVED** [review]

File: test/recording.ts (hunk 3 above)
Comment (adam-in-ict) [reply test/recording.ts:16]: To be honest, I just caught it up with what is now in the integration-template and didn't realize I'd pulled those in.

**Pull request merged**